packages/components/tests/dummy/app/templates/components/link/standalone.hbs

```diff
@@ -115,7 +114,10 @@
         <li>trailing</li>
       </ol>
     </dd>
-    <dt>href</dt>
+    <dt>
```

Comment:
[issue/blocker] this is definitely not required. it may be generated via a `@route` argument, see https://design-system-components-hashicorp.vercel.app/components/link/standalone#:~:text=URLs%20and%20routes%20handling